solarforecastarbiter/metrics/deterministic.py

```diff
@@ -209,3 +197,25 @@ def centered_root_mean_square(y_true, y_pred):
     return np.sqrt(np.mean(
         ((y_pred - np.mean(y_pred)) - (y_true - np.mean(y_true))) ** 2
     ))
+
+
+# Add new metrics to this map to map shorthand to function
+_MAP = {
+    'mae': mean_absolute,
+    'mbe': mean_bias,
+    'rmse': root_mean_square,
+    'mape': mean_absolute_percentage,
+    'nrmse': normalized_root_mean_square,
+    's': forecast_skill,
+    'r': pearson_correlation_coeff,
+    'r^2': coeff_determination,
+    'crmse': centered_root_mean_square
+}
+
+__all__ = [str(m) for m in _MAP]
```

Comment:
mapping the keys to `__all__` seems non-standard. Did you mean `list(map(str, _MAP.values()))`? I'm also ok skipping `__all__` entirely unless there's a reason for it in the already existing code.

Reply:
It was already there but I believe it `__all__` is fairly common to use to ensure `import *` shortcut works for what you intend to be imported from the module. David already had this existing but with the functions.  I tend to never use `import *` but I can certainly see how it could be useful in a module with so much that might be imported.

And yes. I  believe you are correct I need to get the values.

Reply:
Oh. Also, I usually see `__all__` handled in `__init__.py` if anywhere, but I was just (trying) to leave it as it already existed but without the duplication.

Reply:
`__all__ = [m.__name__ for m in _MAP.values()]`

seems to work as intended